app/inbound/return/save-settlement.js

```diff
@@ -4,7 +4,8 @@ const getPaymentRequestbyInvoiceNumber = require('./get-payment-request-by-invoi
 const saveSettlement = async (settlement, transaction) => {
   const matchedPaymentRequest = await getPaymentRequestbyInvoiceNumber(settlement.invoiceNumber, transaction)
   settlement.paymentRequestId = (matchedPaymentRequest ? matchedPaymentRequest?.paymentRequestId : null) ?? null
-  await db.settlement.create(settlement, { transaction })
+  const savedSettlement = await db.settlement.create(settlement, { transaction })
+  await db.schedule.create({ settlementId: savedSettlement.settlementId }, { transaction })
```

Comment:
can we split this out into a sep. func and ensure it is called in `processReturnSettlement` test